SPECS/auoms/auoms.spec

```diff
@@ -53,8 +53,8 @@ cp %{SOURCE3} ./
 %build
 grep AUOMS_BUILDVERSION auoms.version | head -n 4 | cut -d'=' -f2 | tr '\n' '.' | sed 's/.$//' | sed 's/^/#define AUOMS_VERSION "/' > auoms_version.h
 sed -i 's/$/"/' auoms_version.h
-cp -R /usr/include/boost /usr/local/include/boost
-mv /usr/include/boost /usr/include/boost148
+cp -R %{_includedir}/boost %{_prefix}/local/include/boost
+mv %{_includedir}/boost %{_includedir}/boost148
```

Comment:
Please don't modify the path for these two lines. The macros use here are related to the location on the final system where the package gets installed, and they are independend of where these files are located for the package build, so they are not the same thing. In theory we might change our macros at some point if we needed a different folder structure (unlikely) or the package might get modified and these changes should be independent of each other.

Reply:
Discussed offline, resolving.

Reply:
Reverted the destination paths, but left the source paths unchanged. The boost package which provides these files uses the %{_includedir} macro so in theory they should change in lockstep.